README.md

```diff
@@ -36,15 +36,40 @@ Run `yarn lint` to run the linter, or run `yarn lint:fix` to run the linter and
-3. Update version in package.json file (e.g. `yarn version --minor --no-git-tag-version`)
-4. Create a pull request targeting the base branch (e.g. `main` or `1.x`)
-5. Code review and QA
-6. Once approved, the PR is squashed & merged
-7. The commit on the base branch is tagged
-8. The tag can be published as needed
+The project follows the same release process as the other libraries in the MetaMask organization. The GitHub Actions [`action-create-release-pr`](https://github.com/MetaMask/action-create-release-pr) and [`action-publish-release`](https://github.com/MetaMask/action-publish-release) are used to automate the release process; see those repositories for more information about how they work.
+
+1. Choose a release version.
+
+- We follow SemVer, so the set of changes being released dictates what the release version will be. Analyze the changes to see whether they include any breaking changes, new features, or deprecations. See [the SemVer specification](https://semver.org/) for more information.
+
+2. If this release is backporting changes onto a previous release, then ensure there is a major version branch for that version (e.g. `1.x` for a `v1` backport release).
+
+- The major version branch should be set to the most recent release with that major version. For example, when backporting a `v1.0.2` release, you'd want to ensure there was a `1.x` branch that was set to the `v1.0.1` tag.
+
+3. Trigger the `workflow_dispatch` event for the `Create Release Pull Request` action to create the release PR.
+
+- You can find the workflow dispatch button by navigating to the "Actions" tab of the repository, and clicking on the "Create Release Pull Request" workflow in the left sidebar. There should be a "Run workflow" dropdown in the top row of the workflow run table.
+- Press "Run workflow" dropdown and choose a base branch, and either a release type (e.g. "major", "minor", or "patch") or release version. Then press the "Run workflow" button.
+- For a backport release, the base branch should be the major version branch that you ensured existed in step 2. For a normal release, the base branch should be the main branch for that repository (which should be the default value).
+- This should trigger the [`action-create-release-pr`](https://github.com/MetaMask/action-create-release-pr) workflow to create the release PR.
+
+4. Update the changelog to move each change entry into the appropriate change category ([See here](https://keepachangelog.com/en/1.0.0/#types) for the full list of change categories, and the correct ordering), and edit them to be more easily understood by users of the package.
+
+- Generally we omit any changes that don't affect consumers of the package, such as lockfile changes or development environment changes. Exceptions may be made for changes that might be of interest despite not having an effect upon the published package (e.g. major test improvements, security improvements, improved documentation, etc.).
```

Comment:
```suggestion
- Generally any changes are omitted that don't affect consumers of the package, such as lockfile changes or development environment changes. Exceptions may be made for changes that might be of interest despite not having an effect upon the published package (e.g. major test improvements, security improvements, improved documentation, etc.).
```

Reply:
Hmm, that doesn't seem to flow right to me. Avoiding using "we" is a good idea though.

How about this?
```suggestion
- Generally any changes that don't affect consumers of the package (e.g. lockfile changes or development environment changes) are omitted. Exceptions may be made for changes that might be of interest despite not having an effect upon the published package (e.g. major test improvements, security improvements, improved documentation, etc.).
```